src/frontend/kojiBridge/index.ts

```diff
@@ -10,7 +10,11 @@ interface PostMessage {
   kojiEventName: string;
   data?: any;
 }
-
+/**
```

Comment:
Just curious if there is a reason for the padded lines here?

Reply:
Not really. I have been testing various things to see how it affects the output. We can standardize on whatever works best for the parser and for reading the comments in the code.